packages/longhorn/package.yaml

```diff
@@ -1,6 +1,6 @@
 url: https://github.com/longhorn/charts.git
 subdirectory: charts/longhorn
-commit: 93f250e4a175044d34c6982ea6168b213caeee9f
+commit: ec1d661de72966b298833c9533089824bb911696
 version: 100.0.0
```

Comment:
Not sure if I need to bump the version here.

Reply:
no, you do not.  100.0.0 is the version will be released

Reply:
Thanks for your clarification.